index.html

```diff
@@ -38,7 +38,36 @@
             />
```

Comment:
Please add files and folders like `vscode` to `gitignore`.

Reply:
This issue has been resolved